site/en/docs/extensions/mv3/match_patterns/index.md

```diff
@@ -3,154 +3,69 @@ layout: 'layouts/doc-post.njk'
-- [`"externally_connectable.matches"`][ext-connect]
-
-## What is a match pattern? {: #what }
-
-A match pattern is a URL that begins with a permitted scheme, separated by a `://`, followed by a host and path. It can contain wildcard (`*`) characters. Most match patterns are composed of three parts:
+A match pattern is a URL with the following structure, used to specify a group of URLs:
 
 ```text
 &lt;scheme&gt;://&lt;host&gt;/&lt;path&gt;
 ```
 
-Each part can use wildcards `*`. Below is a detailed description:
+**scheme**: Must be one of `'http'`, `'https'`, `'file'`, `'ftp'`, or `'urn'`. The wildcard `*` only matches `http` or `https`.
+  - For information on injecting content scripts into unsupported schemes, such as `about:` and `data:`, see [Injecting in related frames][cs-frames].
+  - The scheme must be separated from the rest of the pattern using a double slash (`//`).
 
-- **scheme**: Must include a valid scheme: `'http'`, `'https'`, `'file'`, `'ftp'`, or `'urn'`. The wildcard `*` only matches `http` or `https`.
+**host**: A hostname (`www.example.com`). A `*` before the hostname to match subdomains (`*.example.com`), or just a wildcard `*`.
+  - If you use a wildcard in the host pattern, it must be the first or only character, and it must be followed by a period (`.`) or forward slash (`/`).
 
-- **host**: A hostname (`www.example.com`), a `*` before the hostname to match subdomains (`*.example.com`), or just a wildcard `*`.
+**path**: Must contain at least a forward slash. The slash by itself is treated as a wildcard `/*`. For example:
+* `/` or `/*`
+* `/foo*`
+* `/foo/bar`
 
-- **path**: Must start with `/` and be present. If alone, it is treated as `/*`. For example: `/*`, `/foo*`, or `/foo/bar`. Each '`*`' matches 0 or more characters.
+Extensions use match patterns for a variety of purposes, including declaring [host permissions][host-permissions] and other permissions for Chrome APIs or [content script][content-scripts] injection, as well as [`"web_accessible_resources.matches"`][war] and [`"externally_connectable.matches"`][ext-connect].
 
 ## Special cases {: #special }
 
-`<all_urls>`
-: It matches any URL that starts with a permitted scheme, including any pattern listed under [valid patterns](#valid-examples). This is a broad host permission, meaning the Chrome web store [review may take longer](/docs/webstore/review-process/#review-time-factors).
+`"<all_urls>"`
+: Matches any URL that starts with a permitted scheme, including any pattern listed under [valid patterns](#valid-examples). Because it affects all hosts, Chrome web store reviews for extensions that use it [may take longer](/docs/webstore/review-process/#review-time-factors).
 
 `file:///`
-: Allows your extension to run on local files. It has no host and requires the user to manually [grant access][permissions-allow-access].
+: Allows your extension to run on local files. This pattern requires the user to manually [grant access][permissions-allow-access].
```

Comment:
```suggestion
: Allows your extension to run on local files. This pattern requires the user to manually [grant access][permissions-allow-access]. Note that this case requires three slashes, not two.
```

Reply:
This link is broken. Please add this to the list at the bottom:

[permissions-allow-access]: /docs/extensions/mv3/declare_permissions/#allow_access

Reply:
I think I've fixed it. Anything else I need to add?